client/signup/steps/domains/index.jsx

```diff
@@ -432,7 +456,10 @@ export class RenderDomainsStep extends Component {
 		);
 
 		this.props.setDesignType( this.getDesignType() );
-		this.props.goToNextStep();
+
+		if ( migrateSite ) {
+			this.props.goToNextStep( 'site-migration' );
+		}
```

Comment:
Shouldn't we still have an `else` for the case where `migrateSite` is false?
```suggestion
		if ( migrateSite ) {
			this.props.goToNextStep( 'site-migration' );
		} else {
			this.props.goToNextStep();
		}
```